app/src/main/java/com/hedvig/app/feature/profile/ui/aboutapp/AboutAppActivity.kt

```diff
@@ -63,11 +67,17 @@ class AboutAppActivity : BaseActivity(R.layout.activity_about_app) {
             versionNumber.text =
                 resources.getString(R.string.PROFILE_ABOUT_APP_VERSION, BuildConfig.VERSION_NAME)
 
-            profileViewModel.data.observe(this@AboutAppActivity) { data ->
-                data.getOrNull()?.member?.id?.let { id ->
-                    memberId.text = resources.getString(R.string.PROFILE_ABOUT_APP_MEMBER_ID, id)
+            memberIdViewModel
+                .state
+                .flowWithLifecycle(lifecycle)
+                .onEach { state ->
+                    when (state) {
+                        is MemberIdViewModel.State.Success -> memberId.text = state.id
```

Comment:
Is it intentional to only show the id here without fetching the string res?

Reply:
Great catch - no, this was a mistake 🙈